.circleci/config.yml

```diff
@@ -0,0 +1,121 @@
+version: 2.1
+orbs:
+  node: circleci/node@4.1.0
+
+references:
+  persist-workspace: &persist-workspace
+    persist_to_workspace:
+      root: ~/source
+      paths:
+        - .
+
+  attach-workspace: &attach-workspace
+    attach_workspace:
+      at: .
+
+defaults: &defaults
+  working_directory: ~/source
+  docker:
+    - image: circleci/node:latest
+  resource_class: medium+
+
+jobs:
+  lint-test:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run:
+          name: Install dependencies
+          command: yarn install
+      - run:
+          name: Run linting
+          command: yarn lint
+      - run:
+          name: Run tests
+          command: yarn test
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+      - *persist-workspace
+  build:
+    <<: *defaults
+    steps:
+      - *attach-workspace
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run: yarn install
```

Comment:
why yarn install again? It's already in the lint-test job

Reply:
I was thinking just in case 😛 But, you're correct, it's redundant so I'll remove it.